spacy/tests/vocab_vectors/test_stringstore.py

```diff
@@ -31,7 +31,7 @@ def test_stringstore_from_api_docs(stringstore):
     assert stringstore["banana"] == banana_hash
 
 
-@pytest.mark.parametrize("text1,text2,text3", [(b"Hello", b"goodbye", b"hello")])
+@pytest.mark.parametrize("text1,text2,text3", [("Hello", "goodbye", "hello")])
```

Comment:
Does it make sense to remove these kinds of tests, or check for errors instead?

Reply:
Good idea - I'll rewrite that test to check for errors instead.